ERROR - 2023-07-07 07:18:34 --> Error connecting to the database: mysqli_sql_exception: Access denied for user ''@'localhost' (using password: NO) in C:\xampp\htdocs\Skripsi-2023\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:173
Stack trace:
#0 C:\xampp\htdocs\Skripsi-2023\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(173): mysqli->real_connect('localhost', '', Object(SensitiveParameterValue), '', 3306, '', 0)
#1 C:\xampp\htdocs\Skripsi-2023\vendor\codeigniter4\framework\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\Skripsi-2023\vendor\codeigniter4\framework\system\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\Skripsi-2023\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1700): CodeIgniter\Database\BaseConnection->query('SELECT COUNT(*)...', Array, false)
#4 C:\xampp\htdocs\Skripsi-2023\vendor\codeigniter4\framework\system\Model.php(606): CodeIgniter\Database\BaseBuilder->countAllResults(false)
#5 C:\xampp\htdocs\Skripsi-2023\vendor\codeigniter4\framework\system\BaseModel.php(1188): CodeIgniter\Model->countAllResults(false)
#6 C:\xampp\htdocs\Skripsi-2023\app\Controllers\RapmaFM.php(30): CodeIgniter\BaseModel->paginate(5, 'podcast')
#7 C:\xampp\htdocs\Skripsi-2023\vendor\codeigniter4\framework\system\CodeIgniter.php(934): App\Controllers\RapmaFM->podcast()
#8 C:\xampp\htdocs\Skripsi-2023\vendor\codeigniter4\framework\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\RapmaFM))
#9 C:\xampp\htdocs\Skripsi-2023\vendor\codeigniter4\framework\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\Skripsi-2023\public\index.php(67): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\Skripsi-2023\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#12 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user ''@'localhost' (using password: NO) in C:\xampp\htdocs\Skripsi-2023\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\xampp\htdocs\Skripsi-2023\vendor\codeigniter4\framework\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\Skripsi-2023\vendor\codeigniter4\framework\system\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\Skripsi-2023\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1700): CodeIgniter\Database\BaseConnection->query('SELECT COUNT(*)...', Array, false)
#3 C:\xampp\htdocs\Skripsi-2023\vendor\codeigniter4\framework\system\Model.php(606): CodeIgniter\Database\BaseBuilder->countAllResults(false)
#4 C:\xampp\htdocs\Skripsi-2023\vendor\codeigniter4\framework\system\BaseModel.php(1188): CodeIgniter\Model->countAllResults(false)
#5 C:\xampp\htdocs\Skripsi-2023\app\Controllers\RapmaFM.php(30): CodeIgniter\BaseModel->paginate(5, 'podcast')
#6 C:\xampp\htdocs\Skripsi-2023\vendor\codeigniter4\framework\system\CodeIgniter.php(934): App\Controllers\RapmaFM->podcast()
#7 C:\xampp\htdocs\Skripsi-2023\vendor\codeigniter4\framework\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\RapmaFM))
#8 C:\xampp\htdocs\Skripsi-2023\vendor\codeigniter4\framework\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\Skripsi-2023\public\index.php(67): CodeIgniter\CodeIgniter->run()
#10 C:\xampp\htdocs\Skripsi-2023\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#11 {main}
CRITICAL - 2023-07-07 07:18:34 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user ''@'localhost' (using password: NO)
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1700): CodeIgniter\Database\BaseConnection->query('SELECT COUNT(*) AS `numrows`
FROM `podcast`', [], false)
 3 SYSTEMPATH\Model.php(606): CodeIgniter\Database\BaseBuilder->countAllResults(false)
 4 SYSTEMPATH\BaseModel.php(1188): CodeIgniter\Model->countAllResults(false)
 5 APPPATH\Controllers\RapmaFM.php(30): CodeIgniter\BaseModel->paginate(5, 'podcast')
 6 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\RapmaFM->podcast()
 7 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\RapmaFM))
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\Skripsi-2023\\public\\index.php')
ERROR - 2023-07-07 07:19:31 --> Error connecting to the database: mysqli_sql_exception: Access denied for user ''@'localhost' (using password: NO) in C:\xampp\htdocs\Skripsi-2023\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:173
Stack trace:
#0 C:\xampp\htdocs\Skripsi-2023\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(173): mysqli->real_connect('localhost', '', Object(SensitiveParameterValue), '', 3306, '', 0)
#1 C:\xampp\htdocs\Skripsi-2023\vendor\codeigniter4\framework\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\Skripsi-2023\vendor\codeigniter4\framework\system\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\Skripsi-2023\vendor\codeigniter4\framework\system\Database\BaseConnection.php(1408): CodeIgniter\Database\BaseConnection->query('SHOW TABLES FRO...')
#4 C:\xampp\htdocs\Skripsi-2023\vendor\codeigniter4\framework\system\Database\BaseConnection.php(1427): CodeIgniter\Database\BaseConnection->listTables()
#5 C:\xampp\htdocs\Skripsi-2023\vendor\codeigniter4\framework\system\Database\MigrationRunner.php(768): CodeIgniter\Database\BaseConnection->tableExists('migrations')
#6 C:\xampp\htdocs\Skripsi-2023\vendor\codeigniter4\framework\system\Database\MigrationRunner.php(168): CodeIgniter\Database\MigrationRunner->ensureTable()
#7 C:\xampp\htdocs\Skripsi-2023\vendor\codeigniter4\framework\system\Commands\Database\Migrate.php(84): CodeIgniter\Database\MigrationRunner->latest(NULL)
#8 C:\xampp\htdocs\Skripsi-2023\vendor\codeigniter4\framework\system\CLI\Commands.php(65): CodeIgniter\Commands\Database\Migrate->run(Array)
#9 C:\xampp\htdocs\Skripsi-2023\vendor\codeigniter4\framework\system\CLI\Console.php(37): CodeIgniter\CLI\Commands->run('migrate', Array)
#10 C:\xampp\htdocs\Skripsi-2023\spark(97): CodeIgniter\CLI\Console->run()
#11 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user ''@'localhost' (using password: NO) in C:\xampp\htdocs\Skripsi-2023\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\xampp\htdocs\Skripsi-2023\vendor\codeigniter4\framework\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\Skripsi-2023\vendor\codeigniter4\framework\system\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\Skripsi-2023\vendor\codeigniter4\framework\system\Database\BaseConnection.php(1408): CodeIgniter\Database\BaseConnection->query('SHOW TABLES FRO...')
#3 C:\xampp\htdocs\Skripsi-2023\vendor\codeigniter4\framework\system\Database\BaseConnection.php(1427): CodeIgniter\Database\BaseConnection->listTables()
#4 C:\xampp\htdocs\Skripsi-2023\vendor\codeigniter4\framework\system\Database\MigrationRunner.php(768): CodeIgniter\Database\BaseConnection->tableExists('migrations')
#5 C:\xampp\htdocs\Skripsi-2023\vendor\codeigniter4\framework\system\Database\MigrationRunner.php(168): CodeIgniter\Database\MigrationRunner->ensureTable()
#6 C:\xampp\htdocs\Skripsi-2023\vendor\codeigniter4\framework\system\Commands\Database\Migrate.php(84): CodeIgniter\Database\MigrationRunner->latest(NULL)
#7 C:\xampp\htdocs\Skripsi-2023\vendor\codeigniter4\framework\system\CLI\Commands.php(65): CodeIgniter\Commands\Database\Migrate->run(Array)
#8 C:\xampp\htdocs\Skripsi-2023\vendor\codeigniter4\framework\system\CLI\Console.php(37): CodeIgniter\CLI\Commands->run('migrate', Array)
#9 C:\xampp\htdocs\Skripsi-2023\spark(97): CodeIgniter\CLI\Console->run()
#10 {main}
ERROR - 2023-07-07 07:20:06 --> Error connecting to the database: mysqli_sql_exception: Access denied for user ''@'localhost' (using password: NO) in C:\xampp\htdocs\Skripsi-2023\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:173
Stack trace:
#0 C:\xampp\htdocs\Skripsi-2023\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(173): mysqli->real_connect('localhost', '', Object(SensitiveParameterValue), '', 3306, '', 0)
#1 C:\xampp\htdocs\Skripsi-2023\vendor\codeigniter4\framework\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\Skripsi-2023\vendor\codeigniter4\framework\system\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\Skripsi-2023\vendor\codeigniter4\framework\system\Database\BaseConnection.php(1408): CodeIgniter\Database\BaseConnection->query('SHOW TABLES FRO...')
#4 C:\xampp\htdocs\Skripsi-2023\vendor\codeigniter4\framework\system\Database\BaseConnection.php(1427): CodeIgniter\Database\BaseConnection->listTables()
#5 C:\xampp\htdocs\Skripsi-2023\vendor\codeigniter4\framework\system\Database\MigrationRunner.php(768): CodeIgniter\Database\BaseConnection->tableExists('migrations')
#6 C:\xampp\htdocs\Skripsi-2023\vendor\codeigniter4\framework\system\Database\MigrationRunner.php(168): CodeIgniter\Database\MigrationRunner->ensureTable()
#7 C:\xampp\htdocs\Skripsi-2023\vendor\codeigniter4\framework\system\Commands\Database\Migrate.php(84): CodeIgniter\Database\MigrationRunner->latest(NULL)
#8 C:\xampp\htdocs\Skripsi-2023\vendor\codeigniter4\framework\system\CLI\Commands.php(65): CodeIgniter\Commands\Database\Migrate->run(Array)
#9 C:\xampp\htdocs\Skripsi-2023\vendor\codeigniter4\framework\system\CLI\Console.php(37): CodeIgniter\CLI\Commands->run('migrate', Array)
#10 C:\xampp\htdocs\Skripsi-2023\spark(97): CodeIgniter\CLI\Console->run()
#11 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user ''@'localhost' (using password: NO) in C:\xampp\htdocs\Skripsi-2023\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\xampp\htdocs\Skripsi-2023\vendor\codeigniter4\framework\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\Skripsi-2023\vendor\codeigniter4\framework\system\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\Skripsi-2023\vendor\codeigniter4\framework\system\Database\BaseConnection.php(1408): CodeIgniter\Database\BaseConnection->query('SHOW TABLES FRO...')
#3 C:\xampp\htdocs\Skripsi-2023\vendor\codeigniter4\framework\system\Database\BaseConnection.php(1427): CodeIgniter\Database\BaseConnection->listTables()
#4 C:\xampp\htdocs\Skripsi-2023\vendor\codeigniter4\framework\system\Database\MigrationRunner.php(768): CodeIgniter\Database\BaseConnection->tableExists('migrations')
#5 C:\xampp\htdocs\Skripsi-2023\vendor\codeigniter4\framework\system\Database\MigrationRunner.php(168): CodeIgniter\Database\MigrationRunner->ensureTable()
#6 C:\xampp\htdocs\Skripsi-2023\vendor\codeigniter4\framework\system\Commands\Database\Migrate.php(84): CodeIgniter\Database\MigrationRunner->latest(NULL)
#7 C:\xampp\htdocs\Skripsi-2023\vendor\codeigniter4\framework\system\CLI\Commands.php(65): CodeIgniter\Commands\Database\Migrate->run(Array)
#8 C:\xampp\htdocs\Skripsi-2023\vendor\codeigniter4\framework\system\CLI\Console.php(37): CodeIgniter\CLI\Commands->run('migrate', Array)
#9 C:\xampp\htdocs\Skripsi-2023\spark(97): CodeIgniter\CLI\Console->run()
#10 {main}
ERROR - 2023-07-07 07:49:38 --> Error connecting to the database: mysqli_sql_exception: Access denied for user ''@'localhost' (using password: NO) in C:\xampp\htdocs\Skripsi-2023\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:173
Stack trace:
#0 C:\xampp\htdocs\Skripsi-2023\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(173): mysqli->real_connect('localhost', '', Object(SensitiveParameterValue), '', 3306, '', 0)
#1 C:\xampp\htdocs\Skripsi-2023\vendor\codeigniter4\framework\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\Skripsi-2023\vendor\codeigniter4\framework\system\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\Skripsi-2023\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1700): CodeIgniter\Database\BaseConnection->query('SELECT COUNT(*)...', Array, false)
#4 C:\xampp\htdocs\Skripsi-2023\vendor\codeigniter4\framework\system\Model.php(606): CodeIgniter\Database\BaseBuilder->countAllResults(false)
#5 C:\xampp\htdocs\Skripsi-2023\vendor\codeigniter4\framework\system\BaseModel.php(1188): CodeIgniter\Model->countAllResults(false)
#6 C:\xampp\htdocs\Skripsi-2023\app\Controllers\RapmaFM.php(30): CodeIgniter\BaseModel->paginate(5, 'podcast')
#7 C:\xampp\htdocs\Skripsi-2023\vendor\codeigniter4\framework\system\CodeIgniter.php(934): App\Controllers\RapmaFM->podcast()
#8 C:\xampp\htdocs\Skripsi-2023\vendor\codeigniter4\framework\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\RapmaFM))
#9 C:\xampp\htdocs\Skripsi-2023\vendor\codeigniter4\framework\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\Skripsi-2023\public\index.php(67): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\Skripsi-2023\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#12 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user ''@'localhost' (using password: NO) in C:\xampp\htdocs\Skripsi-2023\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\xampp\htdocs\Skripsi-2023\vendor\codeigniter4\framework\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\Skripsi-2023\vendor\codeigniter4\framework\system\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\Skripsi-2023\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1700): CodeIgniter\Database\BaseConnection->query('SELECT COUNT(*)...', Array, false)
#3 C:\xampp\htdocs\Skripsi-2023\vendor\codeigniter4\framework\system\Model.php(606): CodeIgniter\Database\BaseBuilder->countAllResults(false)
#4 C:\xampp\htdocs\Skripsi-2023\vendor\codeigniter4\framework\system\BaseModel.php(1188): CodeIgniter\Model->countAllResults(false)
#5 C:\xampp\htdocs\Skripsi-2023\app\Controllers\RapmaFM.php(30): CodeIgniter\BaseModel->paginate(5, 'podcast')
#6 C:\xampp\htdocs\Skripsi-2023\vendor\codeigniter4\framework\system\CodeIgniter.php(934): App\Controllers\RapmaFM->podcast()
#7 C:\xampp\htdocs\Skripsi-2023\vendor\codeigniter4\framework\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\RapmaFM))
#8 C:\xampp\htdocs\Skripsi-2023\vendor\codeigniter4\framework\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\Skripsi-2023\public\index.php(67): CodeIgniter\CodeIgniter->run()
#10 C:\xampp\htdocs\Skripsi-2023\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#11 {main}
CRITICAL - 2023-07-07 07:49:38 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user ''@'localhost' (using password: NO)
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1700): CodeIgniter\Database\BaseConnection->query('SELECT COUNT(*) AS `numrows`
FROM `podcast`', [], false)
 3 SYSTEMPATH\Model.php(606): CodeIgniter\Database\BaseBuilder->countAllResults(false)
 4 SYSTEMPATH\BaseModel.php(1188): CodeIgniter\Model->countAllResults(false)
 5 APPPATH\Controllers\RapmaFM.php(30): CodeIgniter\BaseModel->paginate(5, 'podcast')
 6 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\RapmaFM->podcast()
 7 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\RapmaFM))
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\Skripsi-2023\\public\\index.php')
ERROR - 2023-07-07 07:50:06 --> Error connecting to the database: mysqli_sql_exception: Access denied for user ''@'localhost' (using password: NO) in C:\xampp\htdocs\Skripsi-2023\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:173
Stack trace:
#0 C:\xampp\htdocs\Skripsi-2023\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(173): mysqli->real_connect('localhost', '', Object(SensitiveParameterValue), '', 3306, '', 0)
#1 C:\xampp\htdocs\Skripsi-2023\vendor\codeigniter4\framework\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\Skripsi-2023\vendor\codeigniter4\framework\system\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\Skripsi-2023\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1700): CodeIgniter\Database\BaseConnection->query('SELECT COUNT(*)...', Array, false)
#4 C:\xampp\htdocs\Skripsi-2023\vendor\codeigniter4\framework\system\Model.php(606): CodeIgniter\Database\BaseBuilder->countAllResults(false)
#5 C:\xampp\htdocs\Skripsi-2023\vendor\codeigniter4\framework\system\BaseModel.php(1188): CodeIgniter\Model->countAllResults(false)
#6 C:\xampp\htdocs\Skripsi-2023\app\Controllers\RapmaFM.php(55): CodeIgniter\BaseModel->paginate(5, 'newsflash')
#7 C:\xampp\htdocs\Skripsi-2023\vendor\codeigniter4\framework\system\CodeIgniter.php(934): App\Controllers\RapmaFM->newsflash()
#8 C:\xampp\htdocs\Skripsi-2023\vendor\codeigniter4\framework\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\RapmaFM))
#9 C:\xampp\htdocs\Skripsi-2023\vendor\codeigniter4\framework\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\Skripsi-2023\public\index.php(67): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\Skripsi-2023\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#12 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user ''@'localhost' (using password: NO) in C:\xampp\htdocs\Skripsi-2023\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\xampp\htdocs\Skripsi-2023\vendor\codeigniter4\framework\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\Skripsi-2023\vendor\codeigniter4\framework\system\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\Skripsi-2023\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1700): CodeIgniter\Database\BaseConnection->query('SELECT COUNT(*)...', Array, false)
#3 C:\xampp\htdocs\Skripsi-2023\vendor\codeigniter4\framework\system\Model.php(606): CodeIgniter\Database\BaseBuilder->countAllResults(false)
#4 C:\xampp\htdocs\Skripsi-2023\vendor\codeigniter4\framework\system\BaseModel.php(1188): CodeIgniter\Model->countAllResults(false)
#5 C:\xampp\htdocs\Skripsi-2023\app\Controllers\RapmaFM.php(55): CodeIgniter\BaseModel->paginate(5, 'newsflash')
#6 C:\xampp\htdocs\Skripsi-2023\vendor\codeigniter4\framework\system\CodeIgniter.php(934): App\Controllers\RapmaFM->newsflash()
#7 C:\xampp\htdocs\Skripsi-2023\vendor\codeigniter4\framework\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\RapmaFM))
#8 C:\xampp\htdocs\Skripsi-2023\vendor\codeigniter4\framework\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\Skripsi-2023\public\index.php(67): CodeIgniter\CodeIgniter->run()
#10 C:\xampp\htdocs\Skripsi-2023\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#11 {main}
CRITICAL - 2023-07-07 07:50:06 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user ''@'localhost' (using password: NO)
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1700): CodeIgniter\Database\BaseConnection->query('SELECT COUNT(*) AS `numrows`
FROM `newsflash`', [], false)
 3 SYSTEMPATH\Model.php(606): CodeIgniter\Database\BaseBuilder->countAllResults(false)
 4 SYSTEMPATH\BaseModel.php(1188): CodeIgniter\Model->countAllResults(false)
 5 APPPATH\Controllers\RapmaFM.php(55): CodeIgniter\BaseModel->paginate(5, 'newsflash')
 6 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\RapmaFM->newsflash()
 7 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\RapmaFM))
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\Skripsi-2023\\public\\index.php')
ERROR - 2023-07-07 07:50:50 --> Error connecting to the database: mysqli_sql_exception: Access denied for user ''@'localhost' (using password: NO) in C:\xampp\htdocs\Skripsi-2023\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:173
Stack trace:
#0 C:\xampp\htdocs\Skripsi-2023\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(173): mysqli->real_connect('localhost', '', Object(SensitiveParameterValue), '', 3306, '', 0)
#1 C:\xampp\htdocs\Skripsi-2023\vendor\codeigniter4\framework\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\Skripsi-2023\vendor\codeigniter4\framework\system\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\Skripsi-2023\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\Skripsi-2023\vendor\codeigniter4\framework\system\Model.php(268): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\Skripsi-2023\vendor\codeigniter4\framework\system\BaseModel.php(656): CodeIgniter\Model->doFirst()
#6 C:\xampp\htdocs\Skripsi-2023\vendor\myth\auth\src\Authentication\LocalAuthenticator.php(140): CodeIgniter\BaseModel->first()
#7 C:\xampp\htdocs\Skripsi-2023\vendor\myth\auth\src\Authentication\LocalAuthenticator.php(19): Myth\Auth\Authentication\LocalAuthenticator->validate(Array, true)
#8 C:\xampp\htdocs\Skripsi-2023\vendor\myth\auth\src\Controllers\AuthController.php(87): Myth\Auth\Authentication\LocalAuthenticator->attempt(Array, false)
#9 C:\xampp\htdocs\Skripsi-2023\vendor\codeigniter4\framework\system\CodeIgniter.php(934): Myth\Auth\Controllers\AuthController->attemptLogin()
#10 C:\xampp\htdocs\Skripsi-2023\vendor\codeigniter4\framework\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(Myth\Auth\Controllers\AuthController))
#11 C:\xampp\htdocs\Skripsi-2023\vendor\codeigniter4\framework\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\Skripsi-2023\public\index.php(67): CodeIgniter\CodeIgniter->run()
#13 C:\xampp\htdocs\Skripsi-2023\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#14 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user ''@'localhost' (using password: NO) in C:\xampp\htdocs\Skripsi-2023\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\xampp\htdocs\Skripsi-2023\vendor\codeigniter4\framework\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\Skripsi-2023\vendor\codeigniter4\framework\system\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\Skripsi-2023\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\Skripsi-2023\vendor\codeigniter4\framework\system\Model.php(268): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\Skripsi-2023\vendor\codeigniter4\framework\system\BaseModel.php(656): CodeIgniter\Model->doFirst()
#5 C:\xampp\htdocs\Skripsi-2023\vendor\myth\auth\src\Authentication\LocalAuthenticator.php(140): CodeIgniter\BaseModel->first()
#6 C:\xampp\htdocs\Skripsi-2023\vendor\myth\auth\src\Authentication\LocalAuthenticator.php(19): Myth\Auth\Authentication\LocalAuthenticator->validate(Array, true)
#7 C:\xampp\htdocs\Skripsi-2023\vendor\myth\auth\src\Controllers\AuthController.php(87): Myth\Auth\Authentication\LocalAuthenticator->attempt(Array, false)
#8 C:\xampp\htdocs\Skripsi-2023\vendor\codeigniter4\framework\system\CodeIgniter.php(934): Myth\Auth\Controllers\AuthController->attemptLogin()
#9 C:\xampp\htdocs\Skripsi-2023\vendor\codeigniter4\framework\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(Myth\Auth\Controllers\AuthController))
#10 C:\xampp\htdocs\Skripsi-2023\vendor\codeigniter4\framework\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\Skripsi-2023\public\index.php(67): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\Skripsi-2023\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#13 {main}
CRITICAL - 2023-07-07 07:50:50 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user ''@'localhost' (using password: NO)
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `users`
WHERE `username` = :username:
AND `users`.`deleted_at` IS NULL
 LIMIT 1', [...], false)
 3 SYSTEMPATH\Model.php(268): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(656): CodeIgniter\Model->doFirst()
 5 VENDORPATH\myth\auth\src\Authentication\LocalAuthenticator.php(140): CodeIgniter\BaseModel->first()
 6 VENDORPATH\myth\auth\src\Authentication\LocalAuthenticator.php(19): Myth\Auth\Authentication\LocalAuthenticator->validate([...], true)
 7 VENDORPATH\myth\auth\src\Controllers\AuthController.php(87): Myth\Auth\Authentication\LocalAuthenticator->attempt([...], false)
 8 SYSTEMPATH\CodeIgniter.php(934): Myth\Auth\Controllers\AuthController->attemptLogin()
 9 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(Myth\Auth\Controllers\AuthController))
10 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\Skripsi-2023\\public\\index.php')
